modules/fundamental/src/purl/endpoints/version.rs

```diff
@@ -18,6 +18,7 @@ use trustify_common::purl::Purl;
     ),
 )]
 #[get("/v1/purl/version/{key}")]
+/// Retrieve details of a versioned, non-qualified pURL
```

Comment:
I assume these comments show up on the openapi UI?

Reply:
Yep!